libs/wingc/src/jsify.rs

```diff
@@ -26,7 +26,7 @@ use crate::{
 		is_udt_struct_type,
```

Comment:
Missing tests. Here are a few cases I would check:

* calling a static method from the entrypoint
* calling a JSII static method (that doesn't expect `$scope` as the first argument)
* calling a static method from another static method
* calling a static method from a constructor
* calling a static method from a constructor before calling `super()`